tests/Src/TestElement.php

```diff
@@ -19,4 +20,9 @@ public function getType()
     {
         return 'A test element';
     }
+
+    public function canView($member = null)
+    {
+        return parent::canView($member) && $this->Viewable;
```

Comment:
This is testing the behaviour of a stub. Perhaps instead it could implement a real world permission check like “is admin”, then you can log in as a content author in the tests and assert that it’s not viewable